tests/inputs/selectors.scss

```diff
@@ -303,3 +303,13 @@ $end: ' ';
 #{$selector} a .tocnumber #{$end} {
         min-width: 100%;
 }
+
+
+// escaped
+.\31u\$ {clear: left;}
+.\31 u\$ {clear: left;}
+.\31u\24 {  clear: right;}
+.\31 u\24 {  clear: right;}
+.a\31 u\24 {  clear: both;}
+$x: '\\28' + 'xlarge' + '\\29';
+.\31 2u#{$x}, .\31 2u\24#{$x} { width: 100%; clear: none; margin-left: 0; }
```

Comment:
I suggest that we also contribute upstream to have tests covering that in sass-spec, instead of covering them only in our own testsuite (see #179)

Reply:
Done with https://github.com/sass/sass-spec/pull/1600
Let's see how the processus is working.

Reply:
@Cerdic any plan to take their review into account ?